lib/manager/bundler/locked-version.ts

```diff
@@ -0,0 +1,43 @@
+import { isVersion } from '../../versioning/ruby';
+
+const DEP_REGEX = '(?<=\\().*(?=\\))';
```

Comment:
```suggestion
const DEP_REGEX = new RegExp('(?<=\\().*(?=\\))');
```
Otherwise `string.match` will call `new Regex(DEP_REGEX)`, which will create a lot of unnessessary javascript objects.